### PR TITLE
[MM-29500] Migrate changeCSS() to CSS variable in utils/utils.jsx

### PR DESCRIPTION
**Pull request by oliverJurgen**
#### Summary

Migrate ```changeCss('.app*_body #channel*view.channel-view', `background: ${theme.centerChannelBg}`)``` in line 620 to CSS variables. Css applied in `sass/base/_structure.scss`.

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-server/issues/15887

  Jira: https://mattermost.atlassian.net/browse/MM-29500.

If I have to make changes in the PR, please do tell me. Thanks

### CODE REVIEW

**Review by asaadmahmood — CHANGES_REQUESTED** [review]
Please use it like this:
```
.channel-view {
        &#channel-view {
            background: var(--center-channel-bg);
        }
    }
```

Also please see if we need so much specificity or just `.channel_view` works.

File: sass/base/_structure.scss [hunk 1]
```diff
@@ -24,6 +24,12 @@ body {
             overflow: auto;
         }
     }
+
+    &#channel_view {
+        .channel-view {
+            background: var(--center-channel-bg);
```
Comment (calebroseland) [inline sass/base/_structure.scss:30]: Could `background: var(--center-channel-bg);` go to plain `.channel-view` below around :83?

**Review by calebroseland — CHANGES_REQUESTED** [review]
~~Thanks for the PR, @oliverJurgen! Would you mind moving `background` to the plain `.channel-view` towards the bottom?~~

**Comment by oliverJurgen** [comment]
I applied the changes

**Review by asaadmahmood — APPROVED** [review]

**Comment by oliverJurgen** [comment]
@calebroseland Hello, any more changes I should do?

**Comment by calebroseland** [comment]
@oliverJurgen I think it would be a good chance to simplify and put in just `.channel-view` down by :80. If you wouldn't mind, that be great :)

**Comment by oliverJurgen** [comment]
sounds good, will do that :)

**Comment by oliverJurgen** [comment]
please check my latest change @calebroseland  :)

**Review by calebroseland — APPROVED** [review]
Thanks, @oliverJurgen!

**Comment by jasonblais** [comment]
@calebroseland @asaadmahmood is this ready to merge?

**Comment by calebroseland** [comment]
@jasonblais yes, my bad.

**Pull request merged**